pkg/asset/installconfig/kubevirt/client.go

```diff
@@ -59,9 +60,37 @@ var (
 // LoadKubeConfigContent returns the kubeconfig file content
 func LoadKubeConfigContent() ([]byte, error) {
 	loadingRules := clientcmd.NewDefaultClientConfigLoadingRules()
-	kubeConfigFilename := loadingRules.GetDefaultFilename()
+	configOverrides := &clientcmd.ConfigOverrides{}
+	clientConfig := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(loadingRules, configOverrides)
+	rawConfig, err := clientConfig.RawConfig()
+	if err != nil {
+		return nil, err
+	}
+
+	// Remove anything that is not related to the current context from the result rawConfig
+	var currentContextValue *clientcmdapi.Context
+	for k, v := range rawConfig.Contexts {
+		if k != rawConfig.CurrentContext {
+			delete(rawConfig.Contexts, k)
+		} else {
+			currentContextValue = v
+		}
+	}
+	if currentContextValue == nil {
+		return nil, fmt.Errorf("currentContext is not included in rawConfig.Contexts")
+	}
+	for k := range rawConfig.Clusters {
+		if k != currentContextValue.Cluster {
+			delete(rawConfig.Clusters, k)
+		}
+	}
```

Comment:
```suggestion
	if v, ok := rawConfig.Clusters[currentContextValue.Cluster]; ok {
		rawConfig.Clusters = map[string]*clientcmdapi.Cluster{
			currentContextValue.Cluster: v
		}
	}
```
Do a similar thing for AuthInfos, as well.

Reply:
Changed